src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -118,10 +122,10 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
   }
 
   /* Return value here is a tuple: (workflow name, call name) */
-  private def assertCallFqnWellFormed(callFqn: FullyQualifiedName): Future[(String, String)] = {
+  private def assertCallFqnWellFormed(callFqn: FullyQualifiedName): Try[(String, String)] = {
```

Comment:
I'm still for that, but there's been a lot of rebase hell already between this and Scott's branch and I'd like to do that closer to develop.  :smile: